guide/14-deep-learning/how_sr3_works.ipynb

```diff
@@ -0,0 +1,234 @@
+{
```

Comment:
<p>Instead of phrasing this line </p><p>	If there is out-of-memory issue, decrease the number of res blocks.</p><p>Consider this:</p><p>	Number of res blocks can be decreased when 'out-of-memory' issue is encountered</p><p> </p><p>Optimal results are generally obtained, if trained for atleast 200-300 epochs.</p><p> </p><p>Isnt the number of epochs required to get optimal results data dependent? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1664/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>yes, its data dependent also. we can specify that also</p>